src/AssignTransfer/index.js

```diff
@@ -1,20 +1,19 @@
 import React, { Component } from 'react';
```

Comment:
I've got a searchPage component instead of this index file, which keeps the components together in the component folder- don't have a big opinion which one is best